src/components/map/Graph.js

```diff
@@ -51,106 +54,103 @@ const Title = styled.div`
   padding: 5px;
 `;
 
-@inject(app("Filters", "UI"))
-@observer
-class Graph extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      highlight: {x: 0, y: 0},
-      avgSpeed: {x: 0, y: 0},
-    };
-  }
-
-  secondsToTime() {
```

Comment:
There is actually already an existing secondsToTime helper in `helpers/time.js` for this use case.